src/grt/src/GlobalRouter.cpp

```diff
@@ -448,18 +450,25 @@ NetRouteMap GlobalRouter::findRouting(std::vector<Net*>& nets,
 }
 
 void GlobalRouter::estimateRC()
+{
+  std::map<sta::Corner*, std::ostream*> spef_files;
+  estimateRC(spef_files);
+}
+
+void GlobalRouter::estimateRC(std::map<sta::Corner*, std::ostream*>& spef_files)
 {
   // Remove any existing parasitics.
   sta_->deleteParasitics();
 
   // Make separate parasitics for each corner.
   sta_->setParasiticAnalysisPts(true);
 
+  rsz::SpefWriter spef_writer(logger_, sta_, spef_files);
```

Comment:
Instead of instantiate the SpefWriter here, you could have a single spef_writer on resizer and pass it as a pointer to estimateRC. This way, estimateRC don't need to know about the map of Corner to ostream, since it will be encapsulated on resizer.